test/core/transport/chttp2/ping_callbacks_test.cc

```diff
@@ -540,9 +538,9 @@ TEST(PingCallbacksTest, CancelAllCancelsInflightPings) {
       .WillOnce([]() {
         return EventEngine::TaskHandle{123, 456};
       });
-  auto id = callbacks.StartPing(
-      bitgen, Duration::Hours(24), [] { Crash("should not reach here"); },
-      &event_engine);
+  auto id = callbacks.StartPing(bitgen);
```

Comment:
do we have a test that acks the ping before the timeout is started?

Reply:
Added one